gateway-provider-security-pac4j/src/main/java/org/apache/knox/gateway/pac4j/Pac4jMessages.java

```diff
@@ -47,4 +47,8 @@
   @Message( level = MessageLevel.INFO, text =
       "No private key passphrase alias found. Defaulting to master. Exception encountered: {0}")
   void noPrivateKeyPasshraseProvisioned(Exception e);
+
+  @Message( level = MessageLevel.ERROR, text =
+      "No keystore password alias found. Defaulting to master. Exception encountered: {0}")
```

Comment:
Defaulting to master secret

Reply:
The message in the line above reads
```
No private key passphrase alias found. Defaulting to master. Exception encountered: {0}
```

Should I change both?

Reply:
Yea probably.

Reply:
fixing